### ISSUE TITLE
Suggestion: Block Warding

### ISSUE THREAD

**Issue by jawdylawdy** [Mithion/ArsMagica2#1252]
I was recently toying with the Astral Barrier (mostly seeing how it fares as a way to failsafe enderman farms, which it does) and it occurred to me that a similar concept could be applied in a sort of area ward. The basic concept is this: Since keystones can be used to lock blocks within Ars Magica (most of the machines, various keystone items, even player teleportation, etc.), it should stand to reason that you could merge the two concepts and create something new! Perhaps a warding block that can be locked with a keystone combination and given a focus (and etherium) to function. The foci determine how large an area the ward protects (in a cube size e.g. 7x7x7, 9x9x9, 11x11x11), preventing blocks from being broken, entities from being interacted with, etc. by anyone without a matching keystone.

OR they can be used as a sort of hands-free defense mechanism. With the removal of the caster block, we've been lacking in a solid defense system. I don't remember the foci off the top of my head, but perhaps one would knock an intruder back or damage them, one might teleport them away, etc. etc.. 

I know it's a rather complex (and poorly presented) idea, but it would certainly be an interesting function if you ask me.

**Comment by Zsashas** [comment]
Wait, the casters were removed? It's still in the Compendium...

**Comment by jawdylawdy** [comment]
Yes, they've been gone for a while now. I think since the 1.7.10 move, actually (?)

**Comment by TherminatorX** [comment]
A big :+1: from my end on this idea. Magical force fields ftw!
